roles/mongodb/tasks/install.yml

```diff
@@ -114,18 +120,25 @@
     - /var/log/mongodb
     - "{{ mongodb_db_path }}"    # i.e. /library/dbdata/mongodb/
 
-- name: Install /etc/mongod.conf, mongodb.service, /usr/bin/iiab-mongodb-repair-if-no-lock from templates
+- name: Install mongodb.service, /usr/bin/iiab-mongodb-repair-if-no-lock from templates
   template:
     src: "{{ item.src }}"
     dest: "{{ item.dest }}"
     owner: root
     group: root
     mode: "{{ item.mode }}"
   with_items:
-    - { src: 'mongod.conf.j2', dest: "{{ mongodb_conf }}", mode: '0644' }    # i.e. /etc/mongod.conf
     - { src: 'mongodb.service.j2', dest: '/etc/systemd/system/mongodb.service', mode: '0644' }
     - { src: 'iiab-mongodb-repair-if-no-lock.j2', dest: '/usr/bin/iiab-mongodb-repair-if-no-lock', mode: '0755' }
 
+# daemon_reload is used to force systemd to recognize a newly installed .service file
+# restarted here to ensure the port has been changed
+- name: Disable 'mongodb' systemd service - started on demand by sugarizer
+  systemd:
+    name: mongodb
+    daemon_reload: yes
+    enabled: no
```

Comment:
Commenting out Line 140 could allow other (non-Sugarizer) services to enable MongoDB as needed.

(If Sugarizer truly wants/needs MongoDB disabled, making that happen explicitly in sugarizer/tasks/install.yml might be best?)

Reply:
I would suggest should new apps appear that the method used in sugarizer's service file be use:
# Forces mongodb to start up if not started
Requires=mongodb.service
# Wait for mongodb service to run first
After=mongodb.service

Should these new apps not require a service file then one can add the enabled stanza in their own role.